eth/filters/filter.go

```diff
@@ -54,6 +54,8 @@ type Filter struct {
 
 // New creates a new filter which uses a bloom filter on blocks to figure out whether
 // a particular block is interesting or not.
+// MipMaps allow past blocks to be searched much more efficiently, and can be enabled
+// any time a filter includes addresses to filter on.
 func New(backend Backend, useMipMap bool) *Filter {
```

Comment:
I would only mention that the mip map option can't be used with a light client and leave the address remark out. The light client doesn't create the required bloom filters. The code can figure out that his option is only useful for addresses, its not something the client needs to know.

Maybe its better to remove this option and always try to fetch the bloom filter. If that fails fallback to the less efficient method. But that is something for a separate PR.